crispy_forms/tests/test_form_helper.py

```diff
@@ -870,6 +873,16 @@ def test_label_class_and_field_class_bs4_offset_when_horizontal():
     assert '<div class="offset-sm-3 offset-md-4 col-sm-8 col-md-6">' in html
     assert html.count("col-sm-8") == 7
 
+    # Test col-X pattern
+
+    form.helper.label_class = "col-5 col-lg-4"
+    form.helper.field_class = "col-7 col-lg-8"
+    html = render_crispy_form(form)
```

Comment:
This tests both col-xx-yy and col-x patterns. Could we combine test here (row 878-885) with row (865-874)?